master/internal/api_workspace.go

```diff
@@ -551,11 +558,16 @@ func (a *apiServer) DeleteWorkspace(
 		return nil, err
 	}
 
+	modelsExist, _ := a.workspaceHasModels(ctx, req.Id)
```

Comment:
we should almost never ignore errors, from the users perspective if there is a database issue on this query returning a different error message can be a really confusing experience for the user

Reply:
Makes sense; I misunderstood to what degree we wanted to expose SQL errors to the user. Thanks!